app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class ZIPCode(db.Model):
+    __tablename__ = 'ZIPCodes'
+    id = db.Column(db.Integer, primary_key=True)
+    zipcode = db.Column(db.Integer, unique=True)
```

Comment:
I would make zipcode a string.

For example, imagine if the zip began with a 0 (like Maine).